docs/reference/filebeat/configuration-autodiscover.md

```diff
@@ -103,7 +113,11 @@ filebeat.autodiscover:
             - module: redis
               log:
                 input:
-                  type: container
+                  type: filestream
+                  id: container-${data.docker.container.id}
+                  prospector.scanner.symlinks: true
+			      parsers:
+			        - container: ~
```

Comment:
There is a mix of tabs and spaces here, the indentation is very off:
![2025-04-02_15-30](https://github.com/user-attachments/assets/561daac4-542f-406e-8002-c40724878d9b)
```suggestion
                  parsers:
                    - container: ~
```